articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md

```diff
@@ -0,0 +1,150 @@
+![recall.png](recall.png)  
+F-Score: This metric combines precision and recall into a single number using the harmonic mean, striving for a balance between precision and recall. The F1 Score reaches its best value at 1 (perfect precision and recall) and worst at 0.  
+![f1-score.png](f1-score.png)  
+Mean Average Precision (mAP): is a comprehensive metric that calculates the average precision at each point where recall changes, then takes the mean of these average precisions over all queries in a set. It is especially useful for evaluating systems where the order of retrieved images is significant.  
+![ap_map.png](ap_map.png)  
+Where:  
+GTP:denotes the total number of ground truth positives, that is, the number of truth labels/positive samples;  
+n:denotes the number of images involved in the retrieval;  
+rel@k: is a schematic function that is set to 1 when the first retrieval candidate is a similar sample and 0 otherwise.  
+## 3.2 OCR
+As OCR stands for optical character recognition, OCR technology deals with the problem of recognizing all kinds of different characters. Both handwritten and printed characters can be recognized and converted into a machine-readable, digital data format.  
+Think of any kind of serial number or code consisting of numbers and letters that you need digitized. By using OCR you can transform these codes into a digital output. The technology makes use of many different techniques. Put simply, the image taken is processed, the characters extracted, and are then recognized.  
+What OCR does not do is consider the actual nature of the object that you want to scan. It simply.“takes a look” at the characters that you aim to transform into a digital format. For example, if you scan a word it will learn and recognize the letters, but not the meaning of the word.  
+OCR (Optical Character Recognition) typically consists of two crucial steps: Text Detection and Text Recognition.  
+![figure 6 OCR process.png](figure%206%20OCR%20process.png)  
+Fig 6. OCR process  
+### 3.2.1 Text Detection  
+Text detection is the process of identifying and locating the textual regions within an input image or document.This step involves applying specialized models, such as DBNet, CTPN, and EAST, to efficiently and accurately detect the spatial positions of the text.The output of the text detection step is a set of bounding boxes or text region proposals that encapsulate the textual content.  
+**DBNet**   
+DBNet<sup>[7]</sup>employs a differentiable binarization method, which can train the entire network in an end-to-end manner, avoiding the additional post-processing steps required by the traditional proposal-based methods.  
```

Comment:
need a space after </sup>. There are a few other similar places that need a space.